enatega-multivendor-app/src/screens/Main/Main.js

```
@@ -287,6 +290,19 @@ function Main(props) {
     )
   }
 
+  const filterCusinies = () => {
+    if (data !== undefined) {
+      for (let cui of data?.nearByRestaurantsPreview?.restaurants) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)